include/laser_filters/angular_bounds_filter.h

```diff
@@ -99,6 +100,8 @@ namespace laser_filters
 
           }
         }
+        start_time.sec += d.seconds();
+        start_time.nanosec += d.nanoseconds();
```

Comment:
Does `start_time += d` not work? I think that handling seconds and nanoseconds automatically for you is one of the nice part about using Duration objects.

Reply:
Unfortunately I couldn't make `start_time += d` work.

Reply:
Ah, I see the problem. ROS2 has different types for serializing timestamps than for computing with time. Try changing the earlier line from:

```builtin_interfaces::msg::Time start_time = input_scan.header.stamp;```

to

```rclcpp::Time start_time = input_scan.header.stamp;```

Then `start_time` will be an `rclcpp:::Time` type, and `start_time += d;` should work.

Thanks for working on this!